packages/auth/express/auth.js

```diff
@@ -67,13 +72,26 @@ exports.configure = ({
     const user = transformUser(
       await Users.findOne({id})
     )
+
     if (!user) {
       return next('user not found!')
     }
+
     next(null, user)
   })
 
   // Initialise Passport
   server.use(passport.initialize())
   server.use(passport.session())
+
+  server.use(function (req, res, next) {
+    // Check if a user has more than one and let session expire after a shorter period of time
+    if (req.user && userIsInRoles(req.user, specialRoles)) {
+      req.session.cookie.expires =
+        new Date(Date.now() + SHORT_MAX_AGE_IN_MS)
+      req.session.cookie.maxAge = SHORT_MAX_AGE_IN_MS
```

Comment:
1. ~You have to save the session, to be sure this is applied. Maybe you could extract [this](https://github.com/orbiting/backends/blob/app-authenticator/packages/auth/lib/Sessions.js#L35) into a lib and use it here.~
2. ~You basically added another rolling mechanism here (resulting in +1 database queries). You should check if maxAge is not correct and only modify it when needed. I tested this and it doesn't work, because as soon as this middleware doesn't set the maxAge/expire, the rolling mechanism does, and it reset's the expire based on the configured maxAge. I'm not sure what's the best approach. Maybe keep this middleware but disable rolling.~
Never mind, setting it here, without saving, makes the rolling machanism of express-session use your `maxAge`. So everything should be fine.
https://github.com/expressjs/session/blob/master/session/cookie.js#L39